src/style.css

```diff
@@ -0,0 +1,142 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #f6f6f6;
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+  text-align: center;
+  font-weight: 300;
+}
+
+main {
+  padding: 0 10%;
+}
+
+#todo-list-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 25% 0;
+
+  /* padding: 0 20%; */
+  overflow: hidden;
+  cursor: pointer;
+
+  /* width: 60%; */
+  transition: padding 300ms ease-in-out, max-height 500ms ease-in-out;
+  transform: translate3d(0, 0, 0);
+  max-height: 300px;
+  background: #fff;
+  box-shadow:
+    0 4px 8px 0 rgb(0 0 0 / 20%),
+    0 6px 20px 0 rgb(0 0 0 / 19%)
+
+  /*
+  -webkit-transition: padding 300ms ease-in-out, max-height 500ms ease-in-out;
+
+  -webkit-transform: translate3d(0, 0, 0);
+   */
+
+  /* -webkit-perspective: 1000; */
+}
+
+#inner-container {
+  width: 100%;
+  box-sizing: border-box;
+}
+
+#todo-list-section.active {
+  cursor: auto;
+}
+
+.icon {
+  width: 20px;
+  cursor: pointer;
+  float: none;
+  position: absolute;
+  right: 1rem;
+}
+
+.list-header .icon {
+  top: 1rem;
+}
+
+.toDo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+  width: 100%;
+}
+
+#list-header {
+  /* position: relative; */
+
+  /* padding: 0 92px 0 46px; */
+  flex-direction: row;
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻